app/services/iiif_info_service.rb

```diff
@@ -36,6 +38,15 @@ def info
     end
   end
 
+  ##
+  # Trims the provided sizes so that known large undownloadable sizes are not
+  # returned
+  def sizes(sizes)
+    sizes.to_a.filter do |size|
```

Comment:
Is there something besides an array that will come through as the `sizes` param?

Reply:
I think in reality maybe no, but in a test case it could be `nil`

Reply:
An alternative approach might be to return an empty array in that case that sizes is nil.  Realistically it doesn't make a huge difference in our case but might avoid us needing to cast nils to an arrays (as well as save a very small mount of overhead avoiding casting arrays to arrays).

Reply:
But thinking realistically, I'm pretty sure these are all cached like crazy, and we're likely never to see the scale where this sort of optimization is likely to matter.  This code is totally understandable, so happy to :shipit:.